Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Use fixed snapshot of Debian to create a deterministic environment.
 # Snapshot tags can be found at https://hub.docker.com/r/debian/snapshot/tags
-ARG debian_snapshot=sha256:eeed67d1ae0846429668170fdab5e8f7ed884234db1b2c8075471bd8365cf3a7
+ARG debian_snapshot=sha256:89f4b36d2e15a87382699808a2ca96c6da5f1db928eb0a5cd8bfa38e4edcc189
```

Comment:
You still need to call `./scripts/docker_build` and `./scripts/docker_push` though, otherwise this is not really getting used.

Reply:
FYI still need to fix this before submitting.

Reply:
Done; I've added the updated files in a separate commit.